Neos.Flow/Classes/Error/AbstractExceptionHandler.php

```diff
@@ -117,7 +117,7 @@ public function handleException($exception)
     /**
      * Echoes an exception for the web.
      *
-     * @param object $exception \Exception or \Throwable
+     * @param \Throwable $exception
      * @return void
      */
     abstract protected function echoExceptionWeb($exception);
```

Comment:
I guess you didn't add the type hint here for B/C!?

Reply:
I think, there was some reason for not adding it here. Could be b/c, because all classes implementing the abstract method would need to change.